src/OutputSchemaRenderer.php

```diff
@@ -56,27 +56,32 @@ public function __construct(int $format = self::FORMAT_CONSOLE_COLOR)
         // JTI support
         if (isset($constants['PARENT'], $constants['PARENT_KEY'])) {
             $this->addRenderer(...[
-                new PropertyRenderer($constants['PARENT'], 'Children'),
-                new KeysRenderer($constants['PARENT_KEY'], 'STI key', false),
+                new PropertyRenderer($constants['PARENT'], 'Parent'),
+                new KeysRenderer($constants['PARENT_KEY'], 'Parent key', false),
             ]);
         }
 
         // STI support
-        if (isset($constants['CHILDREN'], $constants['DISCRIMINATOR'])) {
-            $this->addRenderer(...[
-                new PropertyRenderer($constants['CHILDREN'], 'Parent'),
-                new KeysRenderer($constants['DISCRIMINATOR'], 'Parent key', false),
-            ]);
+        if (isset($constants['CHILDREN'])) {
+            $this->addRenderer(new PropertyRenderer($constants['CHILDREN'], 'Children'));
+        }
+
+        if (isset($constants['DISCRIMINATOR'])) {
+            $this->addRenderer(new KeysRenderer($constants['DISCRIMINATOR'], 'Discriminator column', false));
```

Comment:
it is not columnn. It is key or field.
I suggest `STI key` as the shortest option.